site/content/in-dev/unreleased/admin-tool.md

```diff
@@ -0,0 +1,153 @@
+```
+
+To unpack and run the distribution, you can use the following command:
+
+```shell
+cd quarkus/admin/build/distributions
+unzip polaris-quarkus-admin-*.zip
+cd polaris-quarkus-admin-*/
+java -jar polaris-quarkus-admin-*-runner.jar --help
+```
+
+To run the Docker image, use the following command:
+
+```shell
+docker run apache/polaris-admin-tool:latest --help
+```
+`
+The basic usage of the Polaris Admin Tool is outlined below:
+
+```
+Usage: polaris-quarkus-admin-runner.jar [-hV] [COMMAND]
+Polaris Admin Tool
+  -h, --help      Show this help message and exit.
+  -V, --version   Print version information and exit.
+Commands:
+  help       Display help information about the specified command.
+  bootstrap  Bootstraps realms and principal credentials.
+  purge      Purge principal credentials.
+```
+
+## Configuration
+
+The Polaris Admin Tool must be executed with the same configuration as the Polaris server. The
+configuration can be done via environment variables or system properties.
+
+At a minimum, it is necessary to configure the Polaris Admin Tool to connect to the same database
+used by the Polaris server. This can be done by setting the following system properties:
+
+```shell
+java -jar quarkus/admin/build/polaris-quarkus-admin-*-runner.jar \
+    -Dpolaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml
+    -Dpolaris.persistence.eclipselink.persistence-unit=polaris
+```
+
+See the [metastore documentation]({{% ref "metastores" %}}) for more information on configuring the
+database connection.
+
+## Bootstrapping Principal Credentials
+
+The `bootstrap` command is used to bootstrap realms and create the necessary principal credentials for the Polaris
+server. This command is idempotent and can be run multiple times without causing any issues.
+
+```shell
+java -jar quarkus/admin/build/polaris-quarkus-admin-*-runner.jar bootstrap --help
+```
+
+The basic usage of the `bootstrap` command is outlined below:
+
+```
+Usage: polaris-quarkus-admin-runner.jar bootstrap [-hV] [-c=<credentials>]...
+       -r=<realms> [-r=<realms>]...
+Bootstraps realms and principal credentials.
+  -c, --credential=<credentials>
+                         Principal credentials to bootstrap. Must be of the
+                           form 'realm,clientId,clientSecret'.
+  -h, --help             Show this help message and exit.
+  -r, --realm=<realms>   The name of a realm to bootstrap.
+  -V, --version          Print version information and exit.
+```
+
+For example, to bootstrap the `realm1` realm and create its root principal credential with the
+client ID `admin` and client secret `admin`, you can run the following command:
+
+```shell
+java -jar quarkus/admin/build/polaris-quarkus-admin-*-runner.jar bootstrap -r realm1 -c realm1,admin,admin
+```
+
+## Purging Principal Credentials
+
+The `purge` command is used to remove realms and principal credentials from the Polaris server. This
```

Comment:
Shall we mention "realms" in the section title too? Could you add a paragraph on the implications of purging a realm?